tts/UI.def.lua

```diff
@@ -124,7 +125,13 @@ UI = {}
 ---@field attributes nil | tts__UIButtonElement_Attributes
 ---@field value nil | string
 
----@shape tts__UIDropdownElement : tts__UIInputElementBase
+---@shape tts__UICellElement : tts__UIElementBase
+
+---@shape tts__UIDefaultsElement_Attributes : tts__UIElementBase_Attributes
+---@field childControlWidth nil | tts__UIElement_Boolean
+---@field childControlHeight nil | tts__UIElement_Boolean
+
+---@shape tts__UIDropdownElement_Attributes: tts__UIElementBase_Attributes
 ---@field onValueChanged tts__UIElement_CallbackFunctionName @Callback name may be optionally followed by "(selectedText)", "(selectedValue)" or "(selectedIndex)". The latter causes the selected Option's index (0-based) to be passed to the callback (but represented as a string). The former two are identical and match the default behavior (no parameter specified) passing the text value of the Option to the callback.
```

Comment:
Technically this could be omitted (post-inheritance), however I think it's valuable to keep around for the description.